warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__rt_https.sql

```diff
@@ -1,30 +1,48 @@
-WITH feed_guideline_index AS (
-    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
+WITH guideline_index AS (
+    SELECT
+        *
+    FROM {{ ref('int_gtfs_quality__guideline_checks_index') }}
+    WHERE check IN ({{ rt_https_trip_updates() }},
+        {{ rt_https_vehicle_positions() }},
+        {{ rt_https_service_alerts() }})
 ),
 
 rt_daily_url_index AS (
-    SELECT * FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+    SELECT DISTINCT
+        dt,
+        string_url,
+        base64_url
+    FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+),
+
+check_start AS (
+    SELECT MIN(dt) AS first_check_date
+    FROM rt_daily_url_index
 ),
 
 int_gtfs_quality__rt_https AS (
     SELECT
-        idx.date,
-        idx.base64_url,
-        idx.feed_type,
-        CASE WHEN idx.feed_type = 'service_alerts' THEN {{ rt_https_service_alerts() }}
-             WHEN idx.feed_type = 'trip_updates' THEN {{ rt_https_trip_updates() }}
-             WHEN idx.feed_type = 'vehicle_positions' THEN {{ rt_https_vehicle_positions() }}
-        END AS check,
-        {{ best_practices_alignment_rt() }} AS feature,
+        idx.* EXCEPT(status),
+        first_check_date,
         CASE
-            WHEN string_url LIKE 'https%' THEN {{ guidelines_pass_status() }}
-            WHEN string_url IS NOT null AND string_url NOT LIKE 'https%' THEN {{ guidelines_fail_status() }}
+        -- check that the row has the right entity + check combo, then assign statuses
+            WHEN (idx.has_rt_url_tu AND check = {{ rt_https_trip_updates() }})
+                OR (idx.has_rt_url_vp AND check = {{ rt_https_vehicle_positions() }})
+                OR (idx.has_rt_url_sa AND check = {{ rt_https_service_alerts() }})
+                   THEN
+                    CASE
+                        WHEN string_url LIKE 'https%' THEN {{ guidelines_pass_status() }}
```

Comment:
nit: case sensitive, same below

technically our URLs could be uppercase or smth